test/e2e-test/pages/common/common.ts

```diff
@@ -174,5 +175,14 @@ module.exports = {
     Then(/^I expect to be redirect back to the task\-list$/, async () => {
       await I.seeInCurrentUrl('/task-list');
     });
+
+    Then(/^I should see the "([^"]*)" page$/, async (key: string) => {
```

Comment:
The whole point of these tests is that someone with no programming experience can easily read the scenarios and understand what is tested or not, thats why we have to use natural language here, using things like 'REASONS_FOR_APPEAL' is in my opinion not natural and also we are not saving that many lines of codes, so dont see the point of using these variables.

Reply:
I disagree with this. If we don't do this we will need to write a method for each page which can then become complicated to maintain and it's a bit more duplication as functionally it is doing the same thing. I don't think `When I visit the "REASONS_FOR_APPEAL" page` differs a lot from `When I visit reasons for appeal` so I believe it shouldn't make it difficult for someone with no programming experience to understand what this is testing.
So I think the use of keys should be encouraged, however, I am open to suggestions we can always change the keys to plain English so instead of 
`"REASONS_FOR_APPEAL"` I could change it to `"reasons for appeal"`.
Let me know what you think.